packages/miniflare/src/plugins/kv/assets.ts

```diff
@@ -0,0 +1,116 @@
+import fs from "fs/promises";
```

Comment:
this is mostly a copy of `/plugin/kv/sites.ts`, but slightly adapted to our Workers + Assets needs